lib/matplotlib/tests/test_datetime.py

```diff
@@ -661,11 +661,26 @@ def test_plot_date(self):
         ax2.plot_date(x_dates, y_ranges)
         ax3.plot_date(x_ranges, y_dates)
 
-    @pytest.mark.xfail(reason="Test for quiver not written yet")
     @mpl.style.context("default")
     def test_quiver(self):
-        fig, ax = plt.subplots()
-        ax.quiver(...)
+        threshold = 10
+        fig, (ax1, ax2, ax3) = plt.subplots(3, 1, layout="constrained")
+
+        x_dates = np.array(
+                    [datetime.datetime(2024, 1, 1+delta) for delta in range(threshold)]
+                )
+        y_dates = np.array(
+                    [datetime.datetime(2024, 2, 1+delta) for delta in range(threshold)]
+                )
+        x_ranges = np.array(range(threshold))
+        y_ranges = np.array(range(threshold))
```

Comment:
This indent is a bit much:
```suggestion
        x_dates = np.array(
            [datetime.datetime(2024, 1, 1+delta) for delta in range(threshold)]
        )
        y_dates = np.array(
            [datetime.datetime(2024, 2, 1+delta) for delta in range(threshold)]
        )
        x_ranges = np.arange(threshold)
        y_ranges = np.arange(threshold)
```